packages/gasket-plugin-nextjs/lib/build.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const { promisify } = require('util');
+const exec = promisify(require('child_process').exec);
+const builder = require('next/dist/build').default;
+
+const { createConfig } = require('./config');
+
+/**
+ * Checks if Next.js Telemetry is enabled. If so, it disables it.
+ *
+ * https://nextjs.org/telemetry#how-do-i-opt-out
+ *
+ * @param {object} logger gasket logger
+ */
+async function checkTelemetryStatus(logger) {
+  const statusRegex = /(?<=Status: ).*./;
+  const { stdout = '' } = await exec('npx next telemetry status');
```

Comment:
is `npx` okay for gasket? will `yarn` users be okay? I know @kinetifex did some work a while back to make sure yarn worked.

Reply:
Really solid point. Will investigate further.